NAS2D/Renderer/Renderer.cpp

```diff
@@ -124,48 +93,30 @@ void Renderer::fadeOut(std::chrono::milliseconds fadeOutTime)
 }
 
 
-/**
- * Gets whether or not a fade is in progress.
- */
 bool Renderer::isFading() const
 {
 	return (mCurrentFadeType != FadeType::None);
 }
 
 
-/**
- * Gets whether the screen is faded or not.
- */
 bool Renderer::isFaded() const
 {
 	return (mCurrentFade == 255.0f);
 }
 
 
-/**
- * Gets a refernece to the callback signal for fade transitions.
- */
 SignalSource<>& Renderer::fadeComplete()
 {
 	return fadeCompleteSignal;
 }
 
 
-/**
- * Gets the center coordinates of the screen.
- */
 Point<int> Renderer::center() const
 {
 	return Point{0, 0} + mResolution / 2;
 }
 
 
-/**
- * Updates the screen.
- *
- * \note	All derived Renderer objects must call Renderer::update()
- *			before performing screen refreshes.
```

Comment:
I'm a bit uncertain about the value of this comment. Or maybe it's because I'm uncertain about the design. Perhaps this relates to the work to extract fading into a separate control, which was never quite completed.